client/src/store/property/propertyThunks.js

```diff
@@ -0,0 +1,22 @@
+import axios from "axios";
+
+import {
+	propertyListRequest,
+	propertyListSuccess,
+	propertyListFail,
+} from "./propertySlice";
+
+export const fetchPropertyList = () => async (dispatch) => {
+	try {
+		dispatch(propertyListRequest());
+		const { data } = await axios.get("/api/v1/properties/all/");
+
+		dispatch(propertyListSuccess(data));
+	} catch (error) {
+		const errorMessage =
+			error.response && error.response.data.message
+				? error.response.data.message
+				: error.message;
```

Comment:
I wasn't certain, but this could be made a bit more concise using the optional chaining and nullish coalescing operators.

```javascript
const errorMessage = error?.response?.data?.message ?? error.message;
```